src/ParcelRegistry.Projections.BackOffice/appsettings.json

```diff
@@ -0,0 +1,40 @@
+﻿{
+  "ConnectionStrings": {
+    "Events": "Server=(localdb)\\mssqllocaldb;Database=EFProviders.InMemory.ParcelRegistry;Trusted_Connection=True;TrustServerCertificate=True;",
+    "BackOffice": "Server=(localdb)\\mssqllocaldb;Database=EFProviders.InMemory.ParcelRegistry;Trusted_Connection=True;TrustServerCertificate=True;",
+    "BackOfficeProjections": "Server=(localdb)\\mssqllocaldb;Database=EFProviders.InMemory.ParcelRegistry;Trusted_Connection=True;TrustServerCertificate=True;",
+    "BackOfficeProjectionsAdmin": "Server=(localdb)\\mssqllocaldb;Database=EFProviders.InMemory.ParcelRegistry;Trusted_Connection=True;TrustServerCertificate=True;"
+  },
+
+  "DataDog": {
+    "Enabled": true,
+    "Debug": false,
+    "ServiceName": "parcel-registry-projections-backoffice-dev"
+  },
+
+  "DistributedLock": {
+    "Region": "eu-west-1",
```

Comment:
Enabled not required?

Reply:
default is true, but would be nice to specify it.

Reply:
Done.